.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: go
+
+go:
+  - 1.5
```

Comment:
You are right! "tip" does the job. You can still update the version or just skip it. Either way is fine.